src/20_Components/amp-bind.html

```diff
@@ -79,6 +88,25 @@
 
   `[property]="expression"`
 
+-->
+
+<!-- #### Text for textContent -->
+<!--
+The text of this &lt;p&gt; will change when the `currentAnimal` variable
+changes.
+-->
+<p [text]="'This is a ' + currentAnimal + '.'">This is a dog.</p>
+
+<!-- #### class for CSS classes -->
+<!--
+The class of this &lt;p&gt; will change when
+either the `allAnimals` or `currentAnimal` implicit state variables change.
+-->
+
+<p [class]="allAnimals[currentAnimal].styleClass">This text will have have a red background color when cat is selected.</p>
+
+<!-- #### Element-specific attributes -->
+<!--
   In this example, the `src` of this &lt;amp-img&gt; element will change when
```

Comment:
I don't really like using the same sentence over and over again. If there is no new information we should make it shorter. For example:
```
Some amp-component provide custom binding targets for `amp-bind`. 

The src URL for videos embedded with `amp-video` can be changed.

`amp-img` URLs can be changed (does this include srcset as well?).
```
Can you also change the descriptions in the previous paragraphs?